CMakeLists.txt

```diff
@@ -2,137 +2,175 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.6)
-        src/AstTranslator.h
-        src/AstTypeAnalysis.cpp
-        src/AstTypeAnalysis.h
-        src/AstType.h
-        src/AstTypes.h
-        src/AstUtils.cpp
-        src/AstUtils.h
-        src/AstVisitor.h
-        src/BinaryConstraintOps.h
-        src/BinaryFunctorOps.h
-        src/BinaryRelation.h
-        src/BlockList.h
-        src/BTree.h
-        src/CompiledIndexUtils.h
-        src/CompiledOptions.h
-        src/CompiledRecord.h
-        src/CompiledRelation.h
-        src/CompiledSouffle.h
-        src/CompiledTuple.h
-        src/ComponentModel.cpp
-        src/ComponentModel.h
-        src/Constraints.h
-        src/DebugReport.cpp
-        src/DebugReport.h
-        src/ErrorReport.h
-        src/Explain.h
-        src/ExplainProvenance.h
-        src/ExplainProvenanceSLD.h
-        src/ExplainTree.h
-        src/FileFormatConverter.h
-        src/Global.cpp
-        src/Global.h
-        src/GraphUtils.h
-        src/gzfstream.h
-        src/htmx86.h
-        src/IndexSetAnalysis.cpp
-        src/IndexSetAnalysis.h
-        src/InlineRelationsTransformer.cpp
-        src/InterpreterContext.h
-        src/Interpreter.cpp
-        src/Interpreter.h
-        src/InterpreterIndex.h
-        src/InterpreterInterface.h
-        src/InterpreterRecords.cpp
-        src/InterpreterRecords.h
-        src/InterpreterRelation.h
-        src/IODirectives.h
-        src/IOSystem.h
-        src/IterUtils.h
-        src/Logger.h
-        src/Macro.h
-        src/MagicSet.cpp
-        src/MagicSet.h
-        src/main.cpp
-        src/ParallelUtils.h
-        src/ParserDriver.cpp
-        src/ParserDriver.h
-        src/parser.hh
-        src/PrecedenceGraph.cpp
-        src/PrecedenceGraph.h
-        src/ProfileEvent.h
-        src/ProvenanceTransformer.cpp
-        src/RamAnalysis.h
-        src/RamCondition.h
-        src/RamNode.h
-        src/RamOperation.cpp
-        src/RamOperation.h
-        src/RamProgram.h
-        src/RamRelation.h
-        src/RamSemanticChecker.h
-        src/RamStatement.h
-        src/RamTransformer.cpp
-        src/RamTransformer.h
-        src/RamTranslationUnit.h
-        src/RamTypes.h
-        src/RamValue.h
-        src/RamVisitor.h
-        src/ReadStreamCSV.h
-        src/ReadStream.h
-        src/ReadStreamSQLite.h
-        src/SignalHandler.h
-        src/souffle2bdd.cpp
-        src/souffle2lb.cpp
-        src/SouffleInterface.h
-        src/souffle_prof.cpp
-        src/SrcLocation.cpp
-        src/SrcLocation.h
-        src/stack.hh
-        src/StringPool.h
-        src/SymbolMask.h
-        src/SymbolTable.h
-        src/Synthesiser.cpp
-        src/Synthesiser.h
-        src/Table.h
-        src/TernaryFunctorOps.h
-        src/Trie.h
-        src/TypeSystem.cpp
-        src/TypeSystem.h
-        src/UnaryFunctorOps.h
-        src/UnionFind.h
-        src/Util.h
-        src/WriteStreamCSV.h
-        src/WriteStream.h
-        src/WriteStreamSQLite.h
+        ./src/CompiledIndexUtils.h
```

Comment:
This is not friendly to maintain if it changes this much. Does it really help IDEs that use it enough to be worth the effort, and the hassle when it is out of sync with the rest of the source?